### PR TITLE
Update errors.php

### LINKED ISSUE #4519: 🐛 Bug Report: Incorrect Error When Creating Duplicate Domain
### 👟 Reproduction steps

1. Create a custom domain for a project
2. Create another custom domain that's the same as in step 1

### 👍 Expected behavior

Error indicates domain already exists

### 👎 Actual Behavior

Error says duplicate ID:

```json
{
    "message": "A Domain with the requested ID already exists.",
    "code": 409,
    "type": "domain_already_exists",
    "version": "dev"
}
```

### 📝 Tasks

1. [ ] Update error:

https://github.com/appwrite/appwrite/blob/6c220467b8cb33e3784dc99a580cc322b370f9f1/app/config/errors.php#L534

2. [ ] Add a test case:

https://github.com/appwrite/appwrite/blob/939f974d19f30a30c204fa8a7108b857e88673e7/tests/e2e/Services/Projects/ProjectsConsoleClientTest.php#L2158-L2203

3. [ ] Create PR w/ a screenshot of the updated error message

### 🎲 Appwrite version

Version 1.0.x

### 💻 Operating system

Linux

### 🧱 Your Environment

_No response_

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by vedant-z**
## What does this PR do?
Fixes #4519 
(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Comment by vedant-z** [comment]
@stnguyen90 Please review this

File: tests/e2e/Services/Projects/ProjectsConsoleClientTest.php [hunk 1]
```diff
@@ -2156,6 +2156,53 @@ public function testGetProjectPlatform($data): array
         $this->assertEquals('com.example.macos', $response['body']['key']);
         $this->assertEquals('', $response['body']['store']);
         $this->assertEquals('', $response['body']['hostname']);
+        
+        public function testCreateProjectDomain($data): array 
+        { 
+         $id = $data['projectId'] ?? '';
```
Comment (stnguyen90) [inline tests/e2e/Services/Projects/ProjectsConsoleClientTest.php:2162]: Looks like there are style problems indicated by the lint errors. Would you please fix them so the CI job passes?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

**Comment by stnguyen90** [comment]
@vedant-z, would you please run the formatter to fix the styling errors?

FYI, I'll need to close this due to inactivity soon.

**Comment by stnguyen90** [comment]
Closing due to inactivity

**Pull request closed without merging**